src/test/java/com/google/api/codegen/testsrc/showcase/showcase_gapic.yaml

```diff
@@ -20,139 +20,3 @@ language_settings:
     package_name: Google\Showcase\V1beta1
   nodejs:
     package_name: showcase.v1beta1
-# The configuration for the license header to put on generated files.
```

Comment:
Why is all this removed?

Reply:
"all this" being the gapic config? Or the license header stuff? 

I converted the config to gapic v2, so I removed extraneous config, and parts of the license header config are unsupported in gapic v2 so I removed the entire thing. Most gapic v2 configs don't have them anways.